sphinx/api/environment-variables.rst

```diff
@@ -225,14 +225,17 @@ Rose Environment Variables
       The name of the host where the :ref:`cylc install <Install-Workflow>` command was
       invoked.
    Provided By
-      * :ref:`Cylc Rose` and :ref:`cylc install <Install-Workflow>`.
+      * :ref:`Cylc Rose`.
 
 .. envvar:: ROSE_SITE
 
-   Description
-      The value of :rose:conf:`rose.conf|site` setting.
-   Provided By
-      * :ref:`Cylc Rose` and :ref:`cylc install <Install-Workflow>`.
+   .. deprecated:: 2.0.0
+
+      This variable is no longer available.
+
+      .. note::
+
+         :ref:`rose-stem` provides this variable.
```

Comment:
This is a bit confusing, is it deprecated or supported? It's actually obsolete.

Perhaps explain that setting it "globally" is no longer supported, however, via `rose-stem` is still supported.